pkg/install/resources.go

```diff
@@ -326,15 +326,15 @@ func AllResources(o *VeleroOptions) *unstructured.UnstructuredList {
 
 	if o.UseNodeAgent {
 		dsOpts := []podTemplateOption{
-			WithAnnotations(o.PodAnnotations),
-			WithLabels(o.PodLabels),
-			WithImage(o.Image),
-			WithResources(o.NodeAgentPodResources),
-			WithSecret(secretPresent),
-			WithServiceAccountName(serviceAccountName),
```

Comment:
We [use these functions](https://github.com/openshift/oadp-operator/blob/94cf689eee35e51d34804393f6291aca21d5a52a/controllers/restic.go#L177-L182) in openshift/oadp-operator. Any way to keep these exported?

Reply:
Sure. I will make some change to ignore this linter warning.